### DISCUSSION TITLE
Issue with next-plugin-storybook

### DISCUSSION THREAD

**Discussion by jonniebigodes** [vercel/next.js#18204] [Help]
I've been working on updating the Storybook documentation to allow a more seamless integration between Next.js and Storybook and came across a plugin that was implemented on your end, more specifically [@next/plugin-storybook](https://github.com/vercel/next.js/tree/canary/packages/next-plugin-storybook). 

I've done some testing with it and it seems that it's working perfectly with version 5.3.x. 

I extended the scope of my tests to include Storybook version 6.0 and it seems that is currently not working. I'm faced with the following error as soon as i issue `yarn storybook`.

![next-plugin-error 23 15 24](https://user-images.githubusercontent.com/22988955/97089022-bd959100-162c-11eb-990e-2f2db4941961.png)

With this i would like to ask the maintainers if it's something that you would be able to work on and provide with an update for Storybook 6.0?

Based on some questions that are being asked in the Storybook repo it seems that it would something worthwhile to allow a more seamless integration for the community.

Feel free to provide feedback

Stay safe

**Comment by kswilliames** [comment]
Hi @jonniebigodes this is a bug in @next/plugin-storybook. It doesn't pass `rewrites` into the config object in `getWebpackConfig`, so throws an error on this line

https://github.com/vercel/next.js/blob/8b4d6246f31105c9e1fc6f62c75b9ab0633b0de9/packages/next/build/webpack-config.ts#L216

**Comment by jonniebigodes** [comment]
@kswilliames thanks for the insights on this. Is it possible to get a fix on this? I would be more than willing to test it. 

Let me know.

Stay safe

**Reply by kswilliames** [reply]
You're welcome :)

I was about to raise a PR when my issue was also converted to a "help" discussion. But it looks like it's been resolved in [this PR](https://github.com/vercel/next.js/pull/18364)

**Reply by jonniebigodes** [reply]
Indeed. I checked and it seems that "jumped the gun" (pardon the bad pun) asking for a fix, i then checked and it seems that a pull request was up. 

But nonetheless thanks for the willingness to jump in and take care of it.

Stay safe
